lgc/patch/NggPrimShader.cpp

```diff
@@ -5491,7 +5491,8 @@ Function *NggPrimShader::createFetchCullingRegister(Module *module) {
     // regOffset = regOffset >> 2
     regOffset = m_builder->CreateLShr(regOffset, 2); // To dword offset
 
-    auto loadPtr = m_builder->CreateGEP(primShaderTablePtr, {m_builder->getInt32(0), regOffset});
+    auto loadPtr = m_builder->CreateGEP(ArrayType::get(m_builder->getInt32Ty(), 256), primShaderTablePtr,
```

Comment:
Can we create local variable for for this array type and use it to define `primShaderTablePtrTy`?